packages/config/src/ProductType.ts

```diff
@@ -0,0 +1,88 @@
+import { ProductPriceTemplate, ProductVariantsTemplate, ProductTemplate } from './enums';
+
+export type ProductType = {
+
+  /**
+   * Product template
+   * Custom string value could be used in you need to have
+   * specific view in widget
+   */
+  template: ProductTemplate | string
+
+  /**
+   * Product price component display options
+   */
+  price: {
+    display: boolean
+		template: ProductPriceTemplate
+  },
+
+  /**
+   * Title component display options 
+   */
+  title: {
+    display: boolean
+    /**
+     * Max number of title lines to be shown
+     * the rest of text will be cropped
+     */
+    lines: number
+  },
+  
+  /**
+   * Description component display options 
+   */
+  description: {
+    display: boolean
+    /**
+     * Max number of title lines to be shown
+     * the rest of text will be cropped
+     */
+    lines: number
+  },
+  
+  /**
+   * Reviews component display options 
+   */
+  reviews: {
+    display: boolean
+  },
+
+  /**
+   * Variants component display options
+   */
+  variants: {
+    display: boolean
+		template: ProductVariantsTemplate
+  },
+  
+  /**
+   * Image component display options
+   */
+  image: {
+    /** 
+     * If number not set images will keep its original aspect
+     * To make lazy loading work smoothly we should preserve aspect ratio
+		 * 0 - by default
+     */
+    aspectRatio: number
+
+    /**
+     * If "true" images will be loaded just when they appear in view
+     */
+    lazy: boolean
+    /**
+     * Screen offset when images will start loading
+     */
+    lazyOffset: number
+  },
+
+  /**
+   * Sticker component display options
+   */
+  stickers: {
```

Comment:
@Bitaru @vgoloviznin did we agree recently that we want to remove current stickers configuration ?

Reply:
@timakoder for now there will be just display option. Styling options will not be included, and text will be moved to translations

Reply:
@Bitaru we can't move text to translations because it's not a translation, the text will be set by the merchant. I think this section need re-thinking in preparation of upgrading the stickers in the dashboard as this configuration is not useful

Reply:
@vgoloviznin by "translations" i mean i18n, which means localization. So its just a place where we keep strings.
For now stickers have just template which is basically sting and we can use `t('stickers.discount', '20%')`. I think its a best way where we keep single place of strings and templates